resources/options_lookup.tsv

```diff
@@ -1668,6 +1683,12 @@ Clothes Washer	"EnergyStar, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.
 Clothes Washer	"EnergyStar, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.07	rated_annual_energy=123.0	annual_cost=9.0	test_date=2007	drum_volume=3.68	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																
 Clothes Washer	"EnergyStar, Cold Only"	ResidentialApplianceClothesWasher	imef=2.07	rated_annual_energy=123.0	annual_cost=9.0	test_date=2007	drum_volume=3.68	cold_cycle=true	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=0.8	mult_hw=0.8	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar More Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=90.0	annual_cost=8.0	test_date=2013	drum_volume=4.5	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 80% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=60.0	annual_cost=6.0	test_date=2013	drum_volume=4.3	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=0.8	mult_hw=0.8	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 100% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=60.0	annual_cost=6.0	test_date=2013	drum_volume=4.3	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.0	mult_hw=1.0	location=auto	plant_loop=auto																																
+Clothes Washer	"EnergyStar Most Efficient, 120% Usage"	ResidentialApplianceClothesWasher	imef=2.92	rated_annual_energy=60.0	annual_cost=6.0	test_date=2013	drum_volume=4.3	cold_cycle=false	thermostatic_control=true	internal_heater=false	fill_sensor=false	mult_e=1.2	mult_hw=1.2	location=auto	plant_loop=auto																																

```

Comment:
The only two values that make a difference in the clothes washer calculation are rated_annual_energy and annual_cost. These values are different so "most" is a better option. imef and drum_volume are only used for the clothes dryer portion. imef for the "most" option could be increased slightly to reflect the lower rated_annual_energy use, but it will impact only the dryer performance.

Reply:
@JLReyna Thanks for the explanation. A slightly reduced imef that is in roughly line with real products would be nice. We could consider including the rated kWh/yr in the option name to give it more prominence...